app/mailers/user_mailer.rb

```diff
@@ -78,4 +78,16 @@ def backup_ready(user, backup)
       mail to: @resource.email, subject: I18n.t('user_mailer.backup_ready.subject')
     end
   end
+
+  def warning(user, warning)
+    @resource = user
+    @warning  = warning
+    @instance = Rails.configuration.x.local_domain
+
+    I18n.with_locale(@resource.locale || I18n.default_locale) do
+      mail to: @resource.email,
+           subject: I18n.t("user_mailer.warning.subject.#{@warning.action}", acct: "#{user.account.username}@#{@instance}"),
```

Comment:
```
"#{user.account.username}@#{@instance}"
```

Kinda surprised this isn't a function?

Reply:
You are right, I forgot about `Account#local_username_and_domain`